examples/Learning.ipynb

```diff
@@ -0,0 +1,755 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Learning ML model on non-aggregated data"
+      ],
+      "metadata": {
+        "id": "Eult4u-UWW-D"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "OzIbbdhdia_s"
+      },
+      "source": [
+        "Welcome to this tutorial!\n",
+        "\n",
+        "We all know that data is required to train the model, and the quality of this data greatly affects the final result. Training a model on non-aggregated data usually leads to a loss in model quality, so the Toloka team has developed special layers for sublimating such data. In this notebook, we will solve this problem using the LSTM model and integrate the CoNAL and crowdlayer layers from Crowd-Kit library!"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Libraries importing"
+      ],
+      "metadata": {
+        "id": "chiZAWvU_bGX"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rgWLac4VjpTv"
+      },
+      "source": [
+        "First of all, let's install and import necessary libraries"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "neS64tr75OZs"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "import pandas as pd\n",
+        "import numpy as np\n",
+        "%pip install crowd-kit\n",
+        "%pip install sentence_transformers\n",
+        "from crowdkit.learning import CoNAL\n",
+        "from crowdkit.learning import CrowdLayer\n",
+        "import torch\n",
+        "import torch.nn as nn\n",
+        "import torch.optim as optim\n",
+        "from torch.utils.data import Dataset, DataLoader\n",
+        "from transformers import AutoTokenizer\n",
+        "from sentence_transformers import SentenceTransformer"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Boosting with cuda"
+      ],
+      "metadata": {
+        "id": "9fIbGnjO_8JJ"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6T8-rLTjj-9F"
+      },
+      "source": [
+        "It's important to boost our calculous speed so use the GPU"
```

Comment:
Please check the grammar. Also, is it really worth a section? Is boosting a correct word there?